remotepin.go

```diff
@@ -0,0 +1,21 @@
+package config
+
+const (
+	PinningTag             = "Pinning"
+	RemoteServicesTag      = "RemoteServices"
+	RemoteServicesSelector = PinningTag + "." + RemoteServicesTag
+)
+
+type Pinning struct {
+	MFSRepinInterval string // in ns, us, ms, s, m, h
+	RemoteServices   map[string]RemotePinningService
+}
+
+type RemotePinningService struct {
+	Api RemotePinningServiceApi
+}
+
+type RemotePinningServiceApi struct {
+	ApiEndpoint string
+	ApiKey      string
```

Comment:
nit: perhaps we can now avoid `Api.Api*`  and rename them to `Endpoint` and `Key`?

Reply:
Done